src/lib/detection_algorithms/interface/main.js

```diff
@@ -0,0 +1,2 @@
+module.exports.Exporter = require('./exporter.js').Exporter
+module.exports.Reviewer = require('./reviewer.js').Reviewer
```

Comment:
So for the change which removes the config.json you'll have here
module.exports.hide = true;